builds/build-test-nightly.yml

```diff
@@ -0,0 +1,81 @@
+trigger: none
+
+pr: none
+
+# Pipeline trigger - this pipeline will run whenever the 'DacFx Nuget - Nightly' pipeline finishes
+resources:
+  pipelines:
+  - pipeline: 'github-build-test-nightly'
```

Comment:
why nightly? pipeline doesn't have schedule info in the yml file right? you might want to run it with different schedule. can we name it something else?

Reply:
I like the name of the pipelines in ADS better than DacFx. because the schedule is not part of the name. they have a PR validation, product validation ...

Reply:
We already have a PR validation pipeline. How does "latest-nuget-build-test" sound?

Reply:
sounds good